src/pages/Home/Home.jsx

```diff
@@ -6,6 +6,31 @@ import { Link } from "react-router-dom";
 import axios from "axios";
 
 export const Home = () => {
+  const [categories, setCategories] = useState([]);
+  const [homeProducts, setHomeProducts] = useState([]);
+  const [error, setError] = useState("");
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data: categories, status } = await axios.get("/api/categories");
```

Comment:
we can separate out both API calls in different methods & then invoke it in useEffect to make it look more cleaner. 
useEffect(() => {
    fetchProducts();
    fetchCategories();
  }, []);